v1.0/openapi3/definitions/AuthenticationType.yaml

```diff
@@ -0,0 +1,6 @@
+title: AuthenticationType
+type: string
+enum:
+  - OTP
+  - QRCODE
```

Comment:
lack of U2F ?

Reply:
That's how it is in the latest Mojaloop documentation file. 
https://github.com/mojaloop/mojaloop-specification/blob/master/Supporting%20Files/fspiop-rest-v1.0-OpenAPI-implementation_openapi3.yaml#L1960

We can make that change but lets leave that for another PR.